packages/caliper-vscode/extension/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "caliper-vscode-extension",
```

Comment:
Since extensions are stored directly on the vscode marketplace, we should use a different naming convention here (not like the other packages for NPM). I'd suggest: `hyperledger.caliper-extension`

Reply:
This type of naming syntax is not acceptable with vscode. I went with hyperledger-caliper-extension